content/en/network_monitoring/devices/snmp_metrics.md

```diff
@@ -172,6 +172,84 @@ snmp_listener:
 {{% /tab %}}
 {{< /tabs >}}
 
+#### Ping
+
+When configured, the SNMP check can also send ICMP pings to your devices. This can be configured for individual as well as autodiscovered devices.
+
+To set up ping with Network Device Monitoring:
+
+1. Install or upgrade the Datadog Agent to v7.52+. For platform specific instructions, see the [Datadog Agent][7] documentation.
+
+2. Edit the `snmp.d/conf.yaml` file in the `conf.d/` folder at the root of your [Agent's configuration directory][3] for individual devices or the [`datadog.yaml`][8] Agent configuration file for autodiscovery. See the [sample snmp.d/conf.yaml][4] for all available configuration options.
+
+3. **Linux Only**: If you're receiving errors when running ping, you may need to configure the integration to send pings using a raw socket. This requires elevated privileges and is done using the agent's system-probe. See `linux.use_raw_socket` agent configuration and system-probe configuration below.
+
+**Note**: For autodiscovery, if the device does not respond to SNMP, we will not ping it.
+
+{{< tabs >}}
+{{% tab "Individual" %}}
+
+- To apply ping settings to all manually configured devices, create ping configuration in the `init_config` section.
+
+	```yaml
+	    init_config:
+	      loader: core  # use core check implementation of SNMP integration. recommended
+	      use_device_id_as_hostname: true  # recommended
```

Comment:
Are these comments necessary? It makes it look like `use_device_id_as_hostname` is recommended specifically for ping configuration, which I think is confusing (unless that's true?). I think it'd be more useful if you only left in the comments on the actual ping bits

Reply:
Good call, I'll remove them